api.json

```diff
@@ -13,7 +13,7 @@
   },
   "servers": [
     {
-      "url": "https://onesignal.com/api/v1"
+      "url": "{{server}}/api/v1"
```

Comment:
Maybe we should leave this canonical one as pointing to production and swap out the token in the build script?